CIP-0071/README.MD

```diff
@@ -0,0 +1,238 @@
+---
+CIP: 0071
+Title: Verified Pool Identity
+Authors: Dennis Mittmann <dennis@iamx.id>
+Status: Active
+Type: Process
+Created: 2022-10-18
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP defines a standard to verify Staking Pools using Decentralized Identifiers (DID)
+
+## Motivation
+
+The verification process of the authenticity of Staking Pools is currently a centralized effort. 
```

Comment:
May you give perhaps a use case / concrete example of what "the verification process" looks like at the moment and who are the central authority doing this? (I am genuinely not aware of any?)

Reply:
Currently CIP6 is kind of a verification scheme: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0006